src/pages/Teams/AddTeam/AddTeam.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Button,
+  FormControl,
+  TextField,
+  Typography,
+} from '@mui/material';
+import GroupAddIcon from '@mui/icons-material/GroupAdd';
+import { createTeam } from '../../../store/actions/teams';
+import { SET_NOTIFICATION } from '../../../store/actions/types';
+
+const AddTeam = () => {
+  const [name, setName] = useState('');
+  const dispatch = useDispatch();
+
+  const addTeam = () => {
+    if (name.trim().length > 0) {
+      dispatch(createTeam(name));
+    } else {
+      dispatch({
+        type: SET_NOTIFICATION,
+        payload: {
+          message: 'Nom de team incorrect !',
```

Comment:
Can be remained in English as we seem to make all the app in English, I still do not know why mdrr